WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -164,7 +164,11 @@ private static Global CreateGlobal(string dataDir, UiConfig uiConfig, Config con
 		private static async Task TerminateApplicationAsync()
 		{
 			var mainViewModel = MainViewModel.Instance;
-			mainViewModel?.ClearStacks();
+
+			if (mainViewModel is { })
```

Comment:
```suggestion
			if (MainViewModel.Instance is { } mainViewModel )
```
Delete the above line.

Reply:
`mainViewModel` is also used later at line 178.

Reply:
I think we can delete the log message there because there is stuff after the terminate call in `Main()` and also there is a `Logger.LogSoftwareStopped("Wasabi");` log too at the end.

Can you confirm this?

Reply:
This says that the GUI was stopped - it is a different message. But you can move the log message inside the cracked and do the pattern.

Reply:
I moved the stack clearing down instead of moving the log message up. Log structure remains the same with this.

Reply:
I changed the order, and removed the `window.Close()` call.